Hippo.Core/Domain/FinancialDetail.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+
+namespace Hippo.Core.Domain
+{
+    public class FinancialDetail
+    {
+        [Key]
+        int Id { get; set; }
+        [StringLength(128)] //Probably doesn't need to be this big...
+        string SlothApiKey { get; set; }
```

Comment:
`FinancialSystemApiKey` or similar.  Also we might not hold the actual key here but a pointer to the key secret... but I'm ok leaving the name *ApiKey.  Same thing for sloth source below.